aws/ecs_cluster.go

```diff
@@ -11,14 +11,63 @@ import (
 	"github.com/gruntwork-io/gruntwork-cli/errors"
 )
 
+// Used in this context to determine if the ECS Cluster is ready to be used & tagged
+// For more details on other valid status values: https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#Cluster
+const activeEcsClusterStatus string = "ACTIVE"
+const provisioningEcsClusterStatus = "PROVISIONING"
```

Comment:
Does the AWS SDK not export these values itself?

Reply:
Just checked and didn't find it.

Reply:
I've been trying to find them in the awsgo sdk - but they're only described in a comment. I considered having an enum over it, but wasn't sure whether it's needed at this point. Thoughts?